tests/Dfe.PlanTech.Application.UnitTests/Questionnaire/Queries/GetSectionQueryTests.cs

```diff
@@ -135,6 +141,7 @@ public async Task GetSectionBySlug_Returns_Section_From_Db()
     [Fact]
     public async Task GetSectionBySlug_Returns_Section_From_Repository_When_Db_Returns_Nothing()
     {
+        Assert.NotNull(FirstSection.InterstitialPage);
```

Comment:
I'm surprised it has flagged all these as errors, I would have expected it to see the readonly definition with a page and not complain, wheres the type narrowing! sad times

Reply:
I planned to just do the `!` but I was worried it would still warn :D so I just did the assertions instead